scheduler/reconcile_util.go

```diff
@@ -557,6 +562,18 @@ func (a allocSet) delayByMaxClientDisconnect(now time.Time) (later []*delayedRes
 	return
 }
 
+// filterByClientStatus returns allocs from the set with the specified client status.
```

Comment:
No big deal but something to look out for in the future when you're cutting up a PR: `filterByClientStatus` got added in 1a6263f676f335dd11d3f478a71da0cd504e362b but is called in 822e63e2134c04b827d33a6d3ef23abf36afa142 which came before it. We'll squash this PR anyways so it doesn't _really_ matter but it makes reviewing commit-by-commit a little trickier for a large PR like this.

Reply:
Ahh thanks.  Totally makes sense.